Engine/Module/ModuleProgram.h

```diff
@@ -18,24 +18,26 @@ class ModuleProgram : public Module
 	{
 		ENABLE_NORMAL_MAP = 1 << 0,
 		ENABLE_SPECULAR_MAP = 1 << 1,
-		ENABLE_RECEIVE_SHADOWS = 1 << 2,
+		ENABLE_LIGHT_MAP = 1 << 2,
+		ENABLE_RECEIVE_SHADOWS = 1 << 3,
 
-		ENABLE_SPRITESHEET = 1 << 3,
+		ENABLE_SPRITESHEET = 1 << 4,
 
-		ENABLE_BILLBOARD_VIEWPOINT_ALIGNMENT = 1 << 4,
-		ENABLE_BILLBOARD_AXIAL_ALIGNMENT = 1 << 5,
+		ENABLE_BILLBOARD_VIEWPOINT_ALIGNMENT = 1 << 5,
+		ENABLE_BILLBOARD_AXIAL_ALIGNMENT = 1 << 6,
 
-		ENABLE_LIQUID_PROPERTIES = 1 << 6,
-		ENABLE_DISSOLVING_PROPERTIES = 1 << 7,
-		ENABLE_MSAA = 1 << 8,
+		ENABLE_LIQUID_PROPERTIES = 1 << 7,
+		ENABLE_DISSOLVING_PROPERTIES = 1 << 8,
+		ENABLE_MSAA = 1 << 9,
 
-		ENABLE_HDR = 1 << 9,
-		ENABLE_REINHARD = 1 << 10,
-		ENABLE_FILMIC = 1 << 11,
-		ENABLE_EXPOSURE = 1 << 12,
+		ENABLE_HDR = 1 << 10,
+		ENABLE_REINHARD = 1 << 11,
+		ENABLE_FILMIC = 1 << 12,
+		ENABLE_EXPOSURE = 1 << 13,
 
-		ENABLE_BLOOM = 1 << 13,
-		ENABLE_CASCADE_VISUALIZATION = 1 << 14
+		ENABLE_BLOOM = 1 << 14,
+		ENABLE_CASCADE_MAPPING = 1 << 15,
+		ENABLE_CASCADE_VISUALIZATION = 1 << 16
```

Comment:
I would change enum class ShaderVariation to enum class ShaderVariation : int64_t, just to avoid issues when we are playing with bits

Reply:
I didn't know that this could be done :o thanks!